entities/dashboard/widget.json

```diff
@@ -0,0 +1,128 @@
+{
+    "type": "object",
+    "properties": {
+        "uid": {
+            "type": "string",
+            "description": "Unique identifier of the dashboardWidget object"
+        },
+        "app_code_name": {
+            "type": "string",
+            "description": "The code name of the app that owns the widget"
+        },
+        "display_name": {
+            "type": "object",
+            "description": "The widget display name in different languages"
+        },
+        "permissions": {
+            "type": "string",
+            "description": "The permissions list required to be able to use this widget, seperated bu comma"
```

Comment:
We decided this would be a single permission, not a comma-separated list. 
Please change to:
"The permission is required to view the widget. If a user does not have this permission, the widget will not show."